patches/chromium/port_autofill_colors_to_the_color_pipeline.patch

```diff
@@ -7,22 +7,46 @@ https://chromium-review.googlesource.com/c/chromium/src/+/1652925 removed colors
 needed in chromium but our autofill implementation uses them.  This patch can be removed if we refactor
 our autofill implementation to work like Chromium's.
 
+diff --git a/content/browser/web_contents/web_contents_view_aura.cc b/content/browser/web_contents/web_contents_view_aura.cc
+index fa3a6620ce84136476d0ac6e705515c13dc612ab..a0aff5ad93e7644211a2c15553b3d0988f380b7b 100644
+--- a/content/browser/web_contents/web_contents_view_aura.cc
++++ b/content/browser/web_contents/web_contents_view_aura.cc
+@@ -899,13 +899,6 @@ bool WebContentsViewAura::IsValidDragTarget(
+   // for the outermost view. Inner `WebContents` will have a
+   // `WebContentsViewChildFrame` so when dragging between an inner
+   // `WebContents` and its embedder the view IDs will be the same.
+-  bool cross_tab_drag =
+-      GetRenderViewHostID(web_contents_->GetRenderViewHost()) !=
+-      drag_start_->view_id;
+-  return cross_tab_drag;
+-  // TODO(https://crbug.com/1266953): There are some known gaps caused by
+-  // comparing `RenderViewHost` IDs, as `RenderViewHost` ID is not really a
+-  // strong signal for page identity.
```

Comment:
This doesn't belong in this patch file. Also, why is it required?

Reply:
Looks like it was introduced in c731103eac664f79bbf3574b7746e10e3e9b49b0 which says:

> https://chromium-review.googlesource.com/c/chromium/src/+/1652925 removed colors
 needed in chromium but our autofill implementation uses them.  This patch can be removed if we refactor our autofill implementation to work like Chromium's.

I'll make an upgrades-followup ticket for this task

Reply:
@ckerr its not that.  The code that @nornagon mentioned shouldn't be in this patch.

Looking at the history it looks like @VerteDinde added this, presumably to address https://chromium-review.googlesource.com/c/chromium/src/+/3510189.  It should have been in a different patch file though.

Reply:
Yep, @jkleinsc is right, I confused the files during a patch rebase - will fix that now and push up 🙇‍♀️

Reply:
@jkleinsc thanks for the clarification, that makes sense :+1:

@VerteDinde :bow: